### PR TITLE
[Exporter.Geneva] Add LoggerProviderBuilder extensions

### PR DESCRIPTION
**Pull request by vishweshbankwar**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs [hunk 1]
```diff
@@ -27,4 +30,102 @@ public static OpenTelemetryLoggerOptions AddGenevaLogExporter(this OpenTelemetry
...
+    /// Adds Geneva exporter to the LoggerProvider.
+    /// </summary>
+    /// <param name="builder"><see cref="LoggerProviderBuilder"/> builder to use.</param>
+    /// <param name="name">Optional name which is used when retrieving options.</param>
+    /// <param name="configureExporter">Optional callback action for configuring <see cref="GenevaExporterOptions"/>.</param>
+    /// <returns>The instance of <see cref="LoggerProviderBuilder"/> to chain the calls.</returns>
+    public static LoggerProviderBuilder AddGenevaLogExporter(
+        this LoggerProviderBuilder builder,
+        string name,
+        Action<GenevaExporterOptions> configureExporter)
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs:61]: nit:

Consider adding `#nullable enable` to this file so you can decorate these...

```csharp
    public static LoggerProviderBuilder AddGenevaLogExporter(
        this LoggerProviderBuilder builder,
        string? name,
        Action<GenevaExporterOptions>? configureExporter)
```

I think the nullable decoration makes it more clear what the optional things are. But it doesn't look like tracing or metrics are decorated either so up to you if you want to take that on separately or not at all 😄

**Review by CodeBlanch — APPROVED** [review]
LGTM

File: src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs (hunk 1 above)
Comment (Kielek) [reply src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs:61]: If possible, you could enable nullable in whole project in separate PR.
Now, only 5 (Geneva, Instana, ElasticsearchClient, GrpcCore, and HTTP) _production_ projects are missing this functionality. Please include also testing projects to such change.

Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs:61]: I will do it as a separate item.

**Comment by CodeBlanch** [comment]
Adding link: #613

**Pull request merged**